src/constants/Route.tsx

```diff
@@ -1,7 +1,7 @@
 export const routePath = {
   PAGE_TOP : "/",
-  PAGE_ONE : "/location1",
-  PAGE_TWO : "/location2",
-  PAGE_THREE : "/location3",
-  PAGE_FOUR : "/location4",
+  PAGE_ONE : "/gallery/1",
```

Comment:
プロパティ名も変えよう
`/1` は ルートの定義で変数を受け取れるので、ここでは不要ですよ
https://reacttraining.com/react-router/web/api/Route/path-string-string
https://reacttraining.com/react-router/web/example/url-params

```suggestion
  GALLERY: "/gallery",
```

Reply:
レビューいただきました箇所の修正を行い、pushいたしました。